### ISSUE TITLE
is Magisk working?

### ISSUE THREAD

**Issue by MarcosPearce** [phhusson/treble_experimentations#1008]
when i install magisk my motog6play will not boot into the system, is there any way to fix this?

**Comment by JegorRysskin** [comment]
Yes use these modified magisk versions: https://sourceforge.net/projects/expressluke-gsis/files/Extras/phh-magisk-ten/

**Comment by Alphitaa** [comment]
Those doesn't work either. The same with magisk from v200

**Comment by MarcosPearce** [comment]
> Yes use these modified magisk versions: https://sourceforge.net/projects/expressluke-gsis/files/Extras/phh-magisk-ten/

tested but unsuccessful

**Comment by phhusson** [comment]
I don't really care about Magisk, I've already reported them issues with easy fix, and got ignored so...
But anyway, give logs, perhaps I have some luck...

**Comment by MarcosPearce** [comment]
> I don't really care about Magisk, I've already reported them issues with easy fix, and got ignored so...
> But anyway, give logs, perhaps I have some luck...

how do i get the logs because when installed it doesn't leave the boot screen

**Comment by AndyCGYan** [comment]
Certain devices wouldn't boot (gets stuck on 1st splash screen) as soon as Magisk is installed - I've seen some Mi devices on Pie vendor and a Nubia Z17S do this as well. Unfortunately, since they haven't even booted into the OS yet, I don't see what logs can be taken. topjohnwu also explicitly marked GSI issues as "wontfix".

**Comment by Alphitaa** [comment]
The only thing I didn't test yet is the Magisk Fixes module. You can download it from Manager, then put -sysover in the name and flash it in TWRP after Magisk and the GSI. (Only on v20x btw, in Pie you just need to flash an old release and will boot. 19.4 and older)

**Comment by MarcosPearce** [comment]
> A única coisa que eu não testei ainda é o módulo Magisk Fixes. Você pode baixá-lo do Manager, colocar -sysover no nome e atualizá-lo no TWRP após o Magisk e o GSI. (Somente na versão 20x btw, no Pie, você só precisa atualizar uma versão antiga e inicializar. 19.4 e anteriores)

I will try this

**Comment by AndyCGYan** [comment]
> Magisk Fixes module

Tried on the aforementioned Nubia Z17S, no luck. I reckon the problem lies in the Magisk-modified boot image.

**Comment by Alphitaa** [comment]
> > Magisk Fixes module
> 
> Tried on the aforementioned Nubia Z17S, no luck. I reckon the problem lies in the Magisk-modified boot image.

Maybe the patched boot image has trouble with some Pie bootloaders? I think I read somewhere that G6 users need to flash an oreo bootloader in order to boot with Magisk but idk in what cases

**Comment by AndyCGYan** [comment]
> Maybe the patched boot image has trouble with some Pie bootloaders? I think I read somewhere that G6 users need to flash an oreo bootloader in order to boot with Magisk but idk in what cases

On the Mi devices I have/had, Q vendor indeed works with Magisk, but it brings its own issues. Z17S on the other hand uses a community Treble solution (MoKee, the device maintainer of which doesn't even open-source), so there's no choice.

**Comment by gokudrik** [comment]
It's dazing that you don't mention problems with Magisk in your treads on xda or here, in release notes. Why I need to try 100 different variations of everything with Magisk to finally come here and read about Magisk "wontfix" problems with GSI. Omg you rock so much, but if you write few words about magisk problem ... this could have saved a lot of time for a lot of people.

**Comment by Alphitaa** [comment]
Okay so Magisk works fine using Oreo vendor and v208, but Oreo is unstable, has choppy sound, glitchy UI and surely something else, didn't test in depth.

Also tried [Magisk Fixes](https://forum.xda-developers.com/android/software/magiskfixes-fixing-magisk-magisk-t3977203) but didn't change anything

**Comment by MrFrozzen** [comment]
When installing Magisk on asus ze620kl bootloop =((
